### PR TITLE
bug(server): change default session to one month

### LINKED ISSUE #248: I want to stay signed into Chronicle between browser restarts
Right now each time a user restarts their browser they have to sign in. This causes trouble for the add-on since it's using the same authentication.

### PR DESCRIPTION
**Pull request by 6a68**
Fixes #248

@nchapman @pdehaan r?

### CODE REVIEW

File: server/config.js [hunk 1]
```diff
@@ -266,9 +266,9 @@ var conf = convict({
     env: 'SESSION_ALLOW_INSECURE_COOKIES'
   },
   server_session_duration: {
-    doc: 'Session duration in milliseconds. Zero, the default, means session cookie.',
+    doc: 'Session duration in milliseconds. Zero means session cookie. We default to one year.',
     format: 'int',
-    default: 0,
+    default: 31556952000,
```
Comment (pdehaan) [inline server/config.js]: LGTM, unless you want to use a constant here, a la
```js
var SEVEN_DAYS_IN_MSEC = 1000 * 60 * 60 * 24 * 7; // 7 days
```

But the `doc` is already pretty darned descriptive.

**Comment by pdehaan** [comment]
Plus see @nchapman's https://github.com/mozilla/chronicle/issues/248#issuecomment-73730118

File: server/config.js (hunk 1 above)
Comment (6a68) [inline server/config.js]: can't hurt to be consistent within that file, good catch

**Comment by nchapman** [comment]
Looks good! Do we want to update the `local.json.example` file as well?

**Comment by 6a68** [comment]
ah, yeah, sure

**Comment by 6a68** [comment]
alright, /me clicks!

**Pull request merged**